README.md

```diff
@@ -20,8 +20,8 @@ Stream allows developers to rapidly deploy scalable feeds and chat messaging wit
 
 For upgrading from V7 to V8, please refer to the [V8 Migration Guide](https://getstream.io/chat/docs/sdk/flutter/guides/migration_guide_8_0/)
 
-## Sample apps and demos 
-Our team maintains a dedicated repository for full fledged sample applications and demos. Consider checking out [GetStream/flutter-samples](https://github.com/GetStream/flutter-samples) to learn more or get started by looking at our latest [Stream Chat demo](https://github.com/GetStream/flutter-samples/tree/main/packages/stream_chat_v1). 
+## Sample apps and demos
+Check out our [sample_app](sample_app) to see Stream Chat's core features in action. For more sample apps and demos, visit [GetStream/flutter-samples](https://github.com/GetStream/flutter-samples).
```

Comment:
Don't we want to archive `GetStream/flutter-samples`? This repo is also mentioned on line 67

Reply:
We will archive the sample app but we still have some extra demo apps in the repo. Do we want to archive them all?

Reply:
The `chatty` app is officially "no longer maintained". The iMessage app could be nice example, but it's based on stream_chat_flutter_core version 3, so would need a bit of work to get up to date.